recipes/julia-conda-packaging/meta.yaml

```diff
@@ -0,0 +1,48 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
```

Comment:
Remove these

Reply:
Done in db247a0caddd5282b18af3331af4c3763d99f4c9